lib/KafkaSSE.js

```diff
@@ -332,7 +332,7 @@ class KafkaSSE {
             // If anything bad happens during operation, emit an SSE error.
             .catch((e) => {
                 // Log and make the error JSON serializable.
-                e = this._error(e);
+                e = this._error(e, 'warn');
```

Comment:
Hm, I wonder if it would be better to add a catch in the _loop function right after the this.sse.send() call to catch this specific error.  The problem is that the HTTP client is disconnected, which will be noticed if the sse.send() call fails, right?  

Here, we just change all potential error to warn level.

Hm, or perhaps better (to keep the this._error handling call in one place here), we can reject with a specific Error type in SSEResponse.js.  I think the rejections happen in the _write function; we could reject there with a specific error type and then vary the log level we use here based on that error type.